package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "donations",
+  "version": "1.0.0",
+  "description": "donations make easy",
```

Comment:
`donations make easy` --> `generate a donations template for your projects`

Reply:
you can change as you want :dancing_men: